swap/src/bin/asb.rs

```diff
@@ -83,20 +84,21 @@ async fn main() -> Result<()> {
             ask_spread,
         } => {
             let bitcoin_wallet = init_bitcoin_wallet(&config, &seed, env_config).await?;
+
             let monero_wallet = init_monero_wallet(&config, env_config).await?;
 
             let bitcoin_balance = bitcoin_wallet.balance().await?;
-            info!("Bitcoin balance: {}", bitcoin_balance);
+            info!(%bitcoin_balance, "Bitcoin balance");
```

Comment:
The field name is also logged everytime. Would be nice to not be so repetitive here. Maybe the log message can say something more general like:

`Initialized Bitcoin wallet` and then a field `balance` so in the end we have:

```
Initialized Bitcoin wallet balance=0.5 BTC
```

Reply:
This seems to be a recurring theme throughout the change to the log messages in this PR so I am not gonna comment on every occurrence.

Reply:
> This seems to be a recurring theme throughout the change to the log messages in this PR so I am not gonna comment on every occurrence.

My reasoning for this was that it simplifies filtering of the logs. 
Assume the following log: 
```json
{
   "bitcoin_balance":"0.1",
   "msg":"Bitcoin balance."
}
```

 you can simply match on the balance directly 

```bash
| jq .bitcoin_balance
```

while if the field is not unique (e.g. balance field shows up for Bitcoin and Monero) you will need to match on the siblings which can get super cumbersome: 

```
select(.msg == "Bitcoin balance.").bitcoin_balance 
```

I find the former one simpler.

Reply:
I think what @bonomat proposes is fine for now.

Reply:
I see the point but I also think there is a better solution here. One where the human-readable logs don't suffer as much.

1. JSON logs are usually not consumed with JQ, but fed into a tool like vector etc. I'd be very surprised if you can't also easily find that specific log line using a simply search for "bitcoin balance" in whatever frontend sits on top of the log database.
2. You can utilize the _target_ of events to give them more context. (Targets can be specified explicitly instead of relying on the implicit target given by the current module name.)
3. In this particular instance, you could make the ledger an field as well if you want to easily match on it.

All I am asking is that we don't make one situation worse in trying to improve another. JSON logs are a good addition and definitely useful. I also very much like the use of structured logs in favor of free-form messages. However, the logs should be of good quality, regardless in how they are formatted.

The other thing that about this particular log event is that the message by itself doesn't make any sense but it should make sense without the attached fields. Hence my suggestion of `Initialized Bitcoin wallet`. The attached key-value pairs should be considered as **additional** information to the message but not as the primary thing we are outputting.

Reply:
To be clear, I am not hung up on the name of the field being `bitcoin_balance` vs `balance`. I just want the log message by itself to be useful, i.e. `info!(%bitcoin_balance, "Initialized Bitcoin wallet");` is totally fine with me. (Not repeating "bitcoin" would obviously be nice but it is much better than a pointless log of "Bitcoin balance").

Reply:
Ah, so the discussion is about the `message`. 
The changes in this PR aim at producing easily machine parsable logs (may it be JQ or other analyzers). I have no strong opinion on the log `message` itself as long as it does not change :D 

In general, if you prefer a different `message`, can you just propose a change please?

Reply:
Changed to `Initialized Bitcoin wallet` and `Initialized Monero wallet`

Reply:
> In general, if you prefer a different message, can you just propose a change please?

In general I try not to because I don't see it as the reviewer's job to do the problem solving.

In this particular case, my first comment was literally a suggestion of what the line can be changed to so I am a bit confused of what else you are expecting?

Reply:
Right. Will keep it in mind. It doesn't play very well with our aspiration of atomic commits so there isn't a whole lot of value in it which means I usually use to it only because it is nicely rendered by GitHub, not because of the utility of being able to apply the change directly.